models.py

```diff
@@ -86,7 +86,7 @@ def from_dict(cls, mw_dict, controller):
         start = cls.str_to_datetime(mw_dict['start'])
         end = cls.str_to_datetime(mw_dict['end'])
         items = mw_dict['items']
-        description = mw_dict['description'] or None
+        description = mw_dict.get('description', None)
```

Comment:
None is the default for `.get()`

```suggestion
        description = mw_dict.get('description')
```

Reply:
Done